ceph.spec.in

```diff
@@ -921,7 +942,35 @@ Provides:	libcephfs2-devel = %{_epoch_prefix}%{version}-%{release}
 Obsoletes:	libcephfs2-devel < %{_epoch_prefix}%{version}-%{release}
 %description -n libcephfs-devel
 This package contains libraries and headers needed to develop programs
-that use Cephs distributed file system.
+that use Ceph distributed file system.
+
+%if 0%{with jaeger}
+%package -n libjaeger
+Summary:	Ceph distributed file system client library
+%if 0%{?suse_version}
+Group:		System/Libraries
+%endif
+Obsoletes:	libjaeger < %{_epoch_prefix}%{version}-%{release}
+Obsoletes:	libthrift < 0.13.0
+Provides: 	libjaegertracing.so.0()(64bit)
```

Comment:
who will be consuming for instance `libjaegertracing.so.0`?

Reply:
the internals of Provide script was not able to discover `libjaegertracing.so.0` explicitly, and hence all the packages that had libjaeger as a dependency were failing with missing dependencies, explicit indication resolved that issue

Reply:
could you be more specific which packages have libjaeger as a dependency?